coss/base/templates/base.html

```diff
@@ -6,6 +6,7 @@
   <head>
     <meta http-equiv="Content-type" content="text/html; charset=utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="google-site-verification" content="EC15_VrYwUHPZ4kIxgTjJL4ARJoXu0fz3L5h62lGpH8">
```

Comment:
In which account is this identity linked?

Reply:
I'll let @hmitsch chime in on that